pandas/core/indexes/base.py

```diff
@@ -2566,8 +2570,14 @@ def _get_level_values(self, level):
             Maximum distance between original and new labels for inexact
             matches. The values of the index at the matching locations most
             satisfy the equation ``abs(index[indexer] - target) <= tolerance``.
+            Tolerance may be a scalar value, which applies the same tolerance
```

Comment:
same. actually let's fix the shared_docs things first in another PR (or here is ok too). too much doc-string duplication.

Reply:
Makes sense. How would I do that though? This (and the 3 others above) is already in a _shared_docs, so this would require another layer of abstraction above _shared_docs.